### PR TITLE
test: address flaky smoke e2e

### LINKED ISSUE #10148: Fix Swaps tests
### What is this about?

The android swaps tests are flakey on the main branch. We should aim to address this ASAP

### Scenario

_No response_

### Design

_No response_

### Technical Details

_No response_

### Threat Modeling Framework

_No response_

### Acceptance Criteria

_No response_

### Stakeholder review needed before the work gets merged

- [ ] Engineering (needed in most cases)
- [ ] Design
- [ ] Product
- [ ] QA (automation tests are required to pass before merging PRs but not all changes are covered by automation tests - please review if QA is needed beyond automation tests)
- [ ] Security
- [ ] Legal
- [ ] Marketing
- [ ] Management (please specify)
- [ ] Other (please specify)

### References

_No response_

### PR DESCRIPTION
**Pull request by cortisiko**
## **Description**

The advanced gas fee tests are flaky on Bitrise. The transactions simulation info modal made the edit gas fee link text outside the viewport. To fix, a scroll to the bottom of the confirmations screen is needed. 

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: e2e/fixtures/fixture-builder.js [hunk 1]
```diff
@@ -719,6 +719,22 @@ class FixtureBuilder {
     return this;
   }
 
+  withSepoliaNetwork() {
```
Comment (cortisiko) [inline e2e/fixtures/fixture-builder.js:722]: adding sepolia to fixtures

File: e2e/pages/Send/TransactionConfirmView.js [hunk 2]
```diff
@@ -40,7 +39,7 @@ class TransactionConfirmationView {
   }
 
   get transactionAmount() {
-    return Matchers.getElementByText(
+    return Matchers.getElementByID(
```
Comment (cortisiko) [inline e2e/pages/Send/TransactionConfirmView.js:42]: correcting selector strategy

File: e2e/pages/Send/TransactionConfirmView.js [hunk 3]
```diff
@@ -81,6 +80,8 @@ class TransactionConfirmationView {
   }
 
   async tapEstimatedGasLink() {
+    await Gestures.swipe(this.transactionAmount, 'up', 'fast');
```
Comment (cortisiko) [inline e2e/pages/Send/TransactionConfirmView.js:83]: the txn simulation info modal was preventing the edit gas link from being visible in the viewport. Swiping up brings the edit gas link within view

File: e2e/resources/networks.e2e.js [hunk 4]
```diff
@@ -85,9 +84,9 @@ const CustomNetworks = {
   },
   Sepolia: {
     providerConfig: {
-      type: 'mainnet',
-      chainId: '11155111',
-      rpcTarget: 'https://sepolia.infura.io/v3/',
+      type: 'rpc',
+      chainId: '0xaa36a7',
+      rpcTarget: `https://sepolia.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161`,
```
Comment (cortisiko) [inline e2e/resources/networks.e2e.js:89]: this RPC came from chain list including the infura id

File: e2e/resources/networks.e2e.js [hunk 5]
```diff
@@ -98,7 +97,7 @@ const CustomNetworks = {
     providerConfig: {
       type: 'rpc',
       chainId: '0x1',
-      rpcUrl: `https://rpc.tenderly.co/fork/${TENDERLY_KEY}`,
+      rpcUrl: `https://rpc.tenderly.co/fork/bbfe5a2e-2426-4512-a5f8-46ce85fe9ad6`,
```
Comment (cortisiko) [inline e2e/resources/networks.e2e.js:100]: making tenderly rpc public

File: e2e/specs/settings/fiat-on-testnets.spec.js [hunk 6]
```diff
@@ -18,7 +18,7 @@ import { WalletViewSelectorsIDs } from '../../selectors/wallet/WalletView.select
 
 const SEPOLIA = CustomNetworks.Sepolia.providerConfig.nickname;
 
-describe(SmokeCore('Fiat On Testnets Setting'), () => {
+describe(SmokeAssets('Fiat On Testnets Setting'), () => {
```
Comment (cortisiko) [inline e2e/specs/settings/fiat-on-testnets.spec.js:21]: add  proper tag to assets team test

**Review by owencraston — APPROVED** [review]

**Review by OGPoyraz — APPROVED** [review]

**Pull request merged**